### ISSUE TITLE
Cannot delete not used package:

### ISSUE THREAD

**Issue by patrikx3** [npm/npm#15859]
Cannot delete not used package:
```npm unpublish corifeus-core-builder --force```

Error:
```text
patrikx3@workstation:~/Projects/patrikx3/server-scripts$ npm unpublish corifeus-core-builder --force
npm WARN using --force I sure hope you know what you are doing.
npm ERR! Linux 4.9.0-1-amd64
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "unpublish" "corifeus-core-builder" "--force"
npm ERR! node v7.6.0
npm ERR! npm  v4.2.0
npm ERR! code E400

npm ERR! Registry returned 400 for DELETE on https://registry.npmjs.org/corifeus-core-builder/-rev/276-7deab5d64a4de9cbf60773a64a777118
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/patrikx3/.npm/_logs/2017-02-24T09_06_34_800Z-debug.log

```

**Comment by KenanY** [comment]
@patrikx3 Please email `support@npmjs.com` for package unpublishing.

**Comment by bsc1780** [comment]
I have got what i thing is it i been running for six hours. Look guys start your headers like you would a book .the secoung line needs three spaces or doubling up on strings is why its lagging .we have to make up for the space we did not creat .ill give examples in an hour .we gonna runbthis today. It eas realya mis on my part. But its fixed hang tight . when i put it in .lets fix any bug get it out we got watchers.thanks guys.
